Sources/RDeviceIdentifier/RDeviceIdentifierKeychain.swift

```diff
@@ -1,27 +1,17 @@
 import Foundation
-#if !canImport(RSDKUtils)
-import RSDKUtilsMain // Required in SPM version
-#endif
 
 // MARK: - RDeviceIdentifierKeychain
 
 struct RDeviceIdentifierKeychain {
-    func getAccessGroups() throws -> (internalAccessGroup: String, defaultAccessGroup: String)? {
-        // First, try to grab the application identifier prefix (=bundle seed it)
-        // and build the access group from it.
-        do {
-            guard let result = try addIfNonExistent(),
-                  let defaultAccessGroup = result[String(kSecAttrAccessGroup)] as? String,
-                  let teamName = defaultAccessGroup.components(separatedBy: ".").first else {
-                return nil
-            }
-            let accessGroup = "\(teamName).\(RDeviceIdentifierConstants.keychainAccessGroup)"
-            return (accessGroup, defaultAccessGroup)
-        } catch let err {
-            throw err
-        }
+    let service: String
+    let accessGroup: String
+
+    init(service: String, accessGroup: String) {
```

Comment:
I think that the init is not needed as it is automatically generated for structs.